plugins/modules/instance.py

```diff
@@ -523,6 +529,16 @@
         suboptions=linode_instance_placement_group_spec,
         description=["A Placement Group to create this Linode under."],
     ),
+    "lke_cluster_id": SpecField(
+        type=FieldType.integer,
+        description="The id of the LKE cluster associated with this Linode, if there is one.",
+    ),
```

Comment:
`lke_cluster_id` seems a read-only field here. We probably don't need to put it as a SpecField.